src/test/scala/com/johnsnowlabs/nlp/annotators/ner/dl/NerDLReader.scala

```diff
@@ -28,13 +28,13 @@ class NerDLReaderTestSpec extends FlatSpec {
     val spark = ResourceHelper.spark
     import spark.implicits._
 
-    val bp = BasicPipeline().pretrained()
+    val bp = PretrainedPipeline("pipeline_basic")
```

Comment:
Should we just call this `basic` instead of `pipeline_basic`?

Reply:
subject to renew, we should create a new list of pipelines we want to have. this specific test is using that one so we might keep it just fbecause